packages/extension/src/worker/findAnalyticsMatch.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies.
+ */
+import type {
+  CookieAnalytics,
+  CookieDatabase,
+} from '../utils/fetchCookieDictionary';
+
+/**
+ *
+ * @param {string} wildcard  Wildcard cookie name.
+ * @param {string} str cookie name to be matched.
+ * @returns {boolean} Flag for match
+ */
+const wildTest = (wildcard: string, str: string): boolean => {
+  const regExp = wildcard.replace(/[.+^${}()|[\]\\]/g, '\\$&'); // regexp escape
+  const result = new RegExp(
+    `^${regExp.replace(/\*/g, '.*').replace(/\?/g, '.')}$`,
+    'i'
+  );
+  return result.test(str); // remove last 'i' above to have case sensitive
+};
+
+const findAnalyticsMatch = (
```

Comment:
Please add description.

Reply:
Done in https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/25/commits/6c73e0b2e467aa0e280ce72e6b30272f76feb034